system_tests/monitoring.py

```diff
@@ -181,6 +183,53 @@ def test_create_and_delete_metric_descriptor(self):
 
         retry_500(descriptor.create)()
         retry_404_500(descriptor.delete)()
+        descriptor.create()
+        retry_404(descriptor.delete)()
+
+        with self.assertRaises(NotFound):
+            descriptor.delete()
+
+    def test_write_point(self):
+        METRIC_TYPE = ('custom.googleapis.com/tmp/system_test_example' +
+                       unique_resource_id())
+        METRIC_KIND = monitoring.MetricKind.GAUGE
+        VALUE_TYPE = monitoring.ValueType.DOUBLE
+        DESCRIPTION = 'System test example -- DELETE ME!'
+        VALUE = 3.14
+
+        client = monitoring.Client()
+        descriptor = client.metric_descriptor(
+            METRIC_TYPE,
+            metric_kind=METRIC_KIND,
+            value_type=VALUE_TYPE,
+            description=DESCRIPTION,
+        )
+
+        descriptor.create()
+        retry_404(descriptor._fetch)(client, METRIC_TYPE)
```

Comment:
I've been digging deeper into this. Let's be clear that this statement should not be necessary! Nevertheless, the test failed without it.

Strictly speaking, it's not actually the right thing to check. Just because `GetMetricDescriptor` succeeds, that doesn't mean that another call to `GetMetricDescriptor` would succeed, or that a call to `CreateTimeSeries` will succeed.

If you feel more comfortable leaving this as is, that's fine, but my suggestion would be to replace it with `time.sleep(5)`.